docs/usage.md

```diff
@@ -19,7 +19,11 @@ You will need to create a samplesheet with information about the samples you wou
 | `ID`                | An incrementing value which acts as a unique number for the given sample                              |
 | `Sample`            | Custom sample name. This entry will be identical for multiple MS runs from the same sample.           |
 | `Condition`         | Additional information of the sample can be defined here.                                             |
-| `ReplicateFileName` | Full path to the MS file. These files have the extentions .raw, .mzML, mzML.gz, .d, .d.tar.gz, .d.zip |
+| `ReplicateFileName` | Full path to the MS file. These files have the extensions .raw, .mzML, mzML.gz, .d, .d.tar.gz, .d.zip |
+| `Fasta`             | Full path to the FASTA file. These files have the extensions .fasta, .fa, .fas, .fna, .faa, .ffn      |
```

Comment:
It would be nice if the example samplesheet provided at `../assetts/samplesheet.tsv` included some samples with the new `Fasta` column.